retromock/src/main/java/co/infinum/retromock/Calls.java

```diff
@@ -170,5 +176,10 @@ public boolean isCanceled() {
     public Request request() {
       return getDelegate().request();
     }
+
+    @Override
+    public Timeout timeout() {
+      return delegate.timeout();
```

Comment:
This is gonna throw if delegate isn't created already. Use getDelegate() that lazily creates delegate if needed.